src/main/java/com/hm/achievement/listener/PlaceholderAchievementListener.java

```diff
@@ -0,0 +1,56 @@
+package com.hm.achievement.listener;
+
+import org.bukkit.entity.Player;
+
+import com.hm.achievement.AdvancedAchievements;
+import com.hm.achievement.api.AdvancedAchievementsAPI;
+import com.hm.achievement.category.MultipleAchievements;
+import com.hm.achievement.category.NormalAchievements;
+
+import me.clip.placeholderapi.external.EZPlaceholderHook;
+
+public class PlaceholderAchievementListener extends EZPlaceholderHook
+{
+	AdvancedAchievements plugin;
```

Comment:
Please add the `private final` modifier to this field. We don't intend to modify it or use it from another class. ^^